policy/modules/services/ntp.te

```diff
@@ -141,8 +141,12 @@ userdom_list_user_home_dirs(ntpd_t)
 ifdef(`init_systemd',`
 	allow ntpd_t ntpd_unit_t:file read_file_perms;
 
+	allow ntpd_t self:process setfscreate;
```

Comment:
> I'm ok with most of these, but I think the systemd_tmpfiles managing the audit log and locallogin creating utmp is too Buildroot specific, and refpolicy doesn't officially support Buildroot/OE/Yocto.

systemd-tmpfiles is asked to create the /var/log/audit/ directory as auditd doesn't do it by itself. Distributions such as Debian or Fedora get this done when installing their packages, which is not possible in Buildroot (there are no packages, /var/log could be in memory, etc).This is not strickly linked to Buildroot, but to its systemd-tmpfile configuration. My understanding then is that the policy should match the systemd-tmpfile configuration, and a generic support for that in the refpolicy is hard to achieve. Is that correct or is there a room for getting such things supported (other that having a DISTRO-like feature)?

The locallogin rules could be used on any system using Busybox, so that one might not be specific to Buildroot.

> However, if you're a buildroot maintainer that will help supporting this going forward, we can start adding a buildroot (or something similar which would cover OE/yocto) distro option to refpolicy and add these rules.

Having a DISTRO-like option in the refpolicy could be one option when we hit too specific rules; another could be to have in-Buildroot modifications to the refpolicy. I think this should be discussed with others and the Buildroot community before going further.

For now, I'm removing the /var/log/audit and utmp related modifications from this PR. The other patches should be useful not only for Buildroot, and the discussion about Buildroot specifics (whether the modifications are specific or not, and if yes how to address that) can be part of a future and dedicated PR.

Thanks!

Reply:
OK will do.